test/jdk/sun/net/www/http/KeepAliveStream/KeepAliveStreamCloseWithWrongContentLength.java

```diff
@@ -103,7 +136,7 @@ public static void main (String[] args) throws Exception {
             }
             is.close();
         } catch (IOException e) {
-            return;
+            throw new RuntimeException (e);
```

Comment:
No need to catch and wrap since main is declared to throw Exception. Just let IOException and NullPointerExcption percolate out of main...

Reply:
percolate <3